.travis.yml

```diff
@@ -0,0 +1,34 @@
+dist: trusty
+sudo: required
+
+env:
+  - V=HEAD URL_BASE=http://ci.bazel.io/job/Bazel/JAVA_VERSION=1.8,PLATFORM_NAME=linux-x86_64/lastSuccessfulBuild/artifact/output/ci/ URL="$URL_BASE`wget -qO- $URL_BASE | grep -oP 'bazel-.*?-installer.sh' | uniq`" FLAGS=''
+  - V=0.3.2 URL=https://github.com/bazelbuild/bazel/releases/download/0.3.2/bazel-0.3.2-installer-linux-x86_64.sh FLAGS=''
+  - V=0.3.1 URL=https://github.com/bazelbuild/bazel/releases/download/0.3.1/bazel-0.3.1-installer-linux-x86_64.sh FLAGS=''
+
```

Comment:
In my observation, you can cache the build artifacts if you remove extracted Bazel binary before storing cache.
https://github.com/yugui/jsonnetunit/blob/master/.travis.yml#L11